### PR TITLE
fix #1047: localize alias button labels

### LINKED ISSUE #1047: German localization of the site has the “Click to copy” and “Disable/Enable email forwarding for this alias” tooltips not translated
#### Build:
- Stage;

#### Affected Platforms:
- Windows 10;

#### Browsers tested on:
- Firefox, latest version;

#### Prerequisites:
- Be signed in into a Firefox Relay account;
- Have a few aliases created;
- Have the browser language set to German;

#### Steps to reproduce:
1. Go to https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/ ;
2. Scroll down to the alias list;
3. Hover mouse over the “Disable/Enable forwarding for this alias” button;
4. Hover mouse over the alias;
5. Observe both tooltips;

#### Expected result:
- The tooltips should be translated to german;

#### Actual result:
- The tooltips are in english;

#### Notes:
- N/A;
localization 1 1
localization 1 2

### PR DESCRIPTION
**Pull request by groovecoder**
https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/6 needs to merge (and kick off the automated submodule update here) before this merges.

### CODE REVIEW

**Comment by groovecoder** [comment]
Now that mozilla-l10n/fx-private-relay-l10n#6 is merged we can merge this one too.

File: privaterelay/templates/profile.html [hunk 1]
```diff
@@ -146,7 +146,7 @@
           </div> 
 
           <div class="c-alias-copy">
-            <button title="Click to copy"
+            <button title="{% ftlmsg 'click-to-copy' %}"
```
Comment (maxxcrawford) [inline privaterelay/templates/profile.html:149]:
```suggestion
            <button title="{% ftlmsg 'profile-label-click-to-copy' %}"
```

File: privaterelay/templates/profile.html [hunk 2]
```diff
@@ -117,7 +117,7 @@
                   <span class="input-error"></span>
                 </div>
 
-                <button title="Click to copy"
+                <button title="{% ftlmsg 'click-to-copy' %}"
```
Comment (maxxcrawford) [inline privaterelay/templates/profile.html:120]: Looking at the merged strings form https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/6, this should actually be `profile-label-click-to-copy`

```suggestion
                <button title="{% ftlmsg 'profile-label-click-to-copy' %}"
```

File: privaterelay/templates/profile.html [hunk 3]
```diff
@@ -256,7 +256,7 @@
           </div> 
 
           <div class="c-alias-copy">
-            <button title="Click to copy"
+            <button title="{% ftlmsg 'click-to-copy' %}"
```
Comment (maxxcrawford) [inline privaterelay/templates/profile.html:259]:
```suggestion
            <button title="{% ftlmsg 'profile-label-click-to-copy' %}"
```

File: privaterelay/templates/profile.html [hunk 4]
```diff
@@ -227,7 +227,7 @@
                   <span class="input-error"></span>
                 </div>
 
-                <button title="Click to copy"
+                <button title="{% ftlmsg 'click-to-copy' %}"
```
Comment (maxxcrawford) [inline privaterelay/templates/profile.html:230]:
```suggestion
                <button title="{% ftlmsg 'profile-label-click-to-copy' %}"
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]

**Review by maxxcrawford — APPROVED** [review]

**Pull request merged**